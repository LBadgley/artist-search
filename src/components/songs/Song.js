import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';

function Song({ songTitle, artist }) {
  return (
    <Link to={`/lyrics/${artist}/${songTitle}`}>
      {songTitle}
    </Link>
  );
}

Song.propTypes = {
  songTitle: PropTypes.string,
  artist: PropTypes.string
};

export default Song;
