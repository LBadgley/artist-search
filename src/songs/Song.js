import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';

function Song({ songId, songTitle }) {
  return (
    <Link to={`/lyrics/${songId}`}>
      {songTitle}
    </Link>
  );
}

Song.propTypes = {
  songId: PropTypes.string,
  songTitle: PropTypes.string,
};

export default Song;
