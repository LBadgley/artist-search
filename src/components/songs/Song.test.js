import React from 'react';
import { shallow } from 'enzyme';
import Song from './Song';

describe('release test', () => {
  it('renders the release', () => {
    const wrapper = shallow(<Song />);
    expect(wrapper).toMatchSnapshot();
  });
});
