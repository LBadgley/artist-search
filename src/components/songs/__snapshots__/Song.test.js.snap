// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`release test renders the release 1`] = `
<Link
  to="/lyrics/undefined/undefined"
/>
`;
